### PR TITLE
add bare docs and readme

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# anycloud
+# AnyCloud
+
+AnyCloud is an Lambda alternative for Node.js that works with any cloud provider. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
```
Comment (dfellis) [inline README.md:3]: My biggest gripe with Lambda was not being sure what version of any lambda endpoint was actually deployed, and the difficulties of automating removal of dead lambda functions. Not sure how to turn that into a simple bullet point, though.

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+- [ ] Public: Production-ready
+
+## How it works
+
+AnyCloud is a serverless framework built on the Alan and Rust programming languages that:
+
+- [x] Automatically scales your http server across many machines based on request load and system stats
+- [x] Works on multiple cloud providers so you are not locked into AWS
+- [x] Runs locally as-is without special configuration of your local dev environment
+- [ ] In-memory distributed datastore
```
Comment (dfellis) [inline README.md:28]: Should we include this right now, especially since we haven't even documented datastore in the Alan docs?

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+- [ ] GCP (coming soon)
+
+You start using Anycloud without signing up.
+
+**Client libraries**
+
+AnyCloud currently supports the current client libraries:
+- [x] Node.js
+- [ ] Deno
+- [ ] Python
```
Comment (dfellis) [inline README.md:46]: Others to add here? Ruby, Java, etc?

File: README.md (hunk 2 above)
Comment (depombo) [reply README.md:28]: yes, it is not checked off so it seems fine by me

File: README.md (hunk 1 above)
Comment (depombo) [reply README.md:3]: hmmm good point

**Review by dfellis — APPROVED** [review]

**Pull request merged**